weasel/src/main/java/org/edmcouncil/spec/fibo/weasel/ontology/data/OwlDataHandler.java

```diff
@@ -398,4 +422,72 @@ public OwlDetails handleParticularObjectProperty(IRI iri, OWLOntology ontology)
     return result;
   }
 
+  public OwlDetailsProperties<PropertyValue> handleParticularSubClassOf(OWLOntology ontology, OWLClass clazz) {
+    OwlDetailsProperties<PropertyValue> result = new OwlDetailsProperties<PropertyValue>();
+
+    Iterator<OWLSubClassOfAxiom> iterator = ontology.subClassAxiomsForSuperClass(clazz).iterator();
+    //List<OWLSubClassOfAxiom> result = new LinkedList<>();
+    while (iterator.hasNext()) {
+      OWLSubClassOfAxiom next = iterator.next();
+      IRI iri = next.getSubClass().asOWLClass().getIRI();
+      String fragment = iri.getFragment();
+      LOGGER.debug(next.toString());
+      OWLClassExpression superClass = next.getSuperClass();
+//      result.addAll(result);
+      OwlDirectedSubClassesProperty r = new OwlDirectedSubClassesProperty();
+      r.setType(WeaselOwlType.DIRECT_SUBCLASSES);
+      r.setValue(new PairImpl(fragment, iri.toString()));
+      result.addProperty(WeaselOwlType.DIRECT_SUBCLASSES.name(), r);
+      // List<OwlGroupedDetails> newResult = new LinkedList<>();
+      // newResult.addAll(newResult);
+    }
+
+    return result;
+  }
+
+  private OwlDetailsProperties<PropertyValue> handleListIndividual(OWLOntology ontology, OWLClass clazz) {
+    OwlDetailsProperties<PropertyValue> result = new OwlDetailsProperties<PropertyValue>();
+    //Iterator<OWLNamedIndividual> iterator = ontology.individualsInSignature().iterator();
+    //List<OWLNamedIndividual> result = new LinkedList<>();
+    OWLReasonerFactory reasonerFactory = new StructuralReasonerFactory();
+    OWLReasoner reasoner = reasonerFactory.createNonBufferingReasoner(ontology);
+    NodeSet<OWLNamedIndividual> instances = reasoner.getInstances(clazz, true);
+    for (OWLNamedIndividual o : instances.entities().collect(Collectors.toSet())) {
+      String fragment = o.getIRI().getFragment();
+      LOGGER.debug(o.getIRI().toString());
```

Comment:
@patrycjamia Change the log to a more transparent for user, this information says nothing and may be misleading.